bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/thermostat/ThermostatHandler.java

```diff
@@ -38,10 +40,12 @@
 public final class ThermostatHandler extends AbstractBatteryPoweredDeviceHandler {
```

Comment:
Do you want to add your contribution as an author?

Reply:
Done, thanks for the hint :+1: